python/pyspark/worker.py

```diff
@@ -46,6 +46,7 @@
 
 if sys.version >= '3':
     basestring = str
+    unicode = str
```

Comment:
This is necessary, see https://github.com/apache/spark/pull/25814#issuecomment-533147556

Reply:
Then I think we don't need the comditionat 603 line